search2/python/search/save/handler.py

```diff
@@ -164,15 +148,15 @@ def _make_output_string(rows, delims):
 
 
 @ray.remote
-def _process_query_chunk(query_args: dict, search_client_args: dict, field_names: list, chunk_output_name: str, reporter):
+def _process_query_chunk(query_args: dict, search_client_args: dict, field_names: list,
```

Comment:
what's the type of reporter?

Reply:
the reporter reports values to the frontend via beanstalkd